packages/cashier/src/components/cashier-locked/cashier-locked.tsx

```diff
@@ -1,9 +1,24 @@
-import PropTypes from 'prop-types';
 import React from 'react';
+import { GetAccountStatus, Authorize } from '@deriv/api-types';
 import { Icon, Text } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
 import { formatDate } from '@deriv/shared';
 import { connect } from 'Stores/connect';
+import { RootStore } from 'Types';
+
+type TAccount = NonNullable<Authorize['account_list']>[0];
```

Comment:
@hamid-deriv It seems to be a shared type and can be used in other places, I suggest moving it to `types.ts`.